### ISSUE TITLE
signal SIGABRT crash at showTakeoverNotificationWithObject

### ISSUE THREAD

**Issue by clkr** [mixpanel/mixpanel-iphone#207]
UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"MPNotification" bundle:nil]; crashes with signal SIGABRT when there is a takeover notification.

Function:
- (BOOL)showTakeoverNotificationWithObject:(MPNotification *)notification

**Comment by arittr** [comment]
@clkr thanks for this - i'm going to see if we can patch this edge case going forward, but for now you might want to just copy over the XIB/resource manually into your app bundle. i believe this is happening because our library assumes the storyboard to be in the app bundle, which isn't always the case - let me know if the manual copy helps at all!

**Comment by clkr** [comment]
@arittr copying manually solves the crash, thanks! I've also noticed that Titanium guys had a similar problem and found a solution, it might help:
https://github.com/hyperlab/TiMixpanel/issues/4

**Comment by arittr** [comment]
@clkr fantastic - so happy we got this fixed. thanks so much for sending this my way - great solution and very well documented. will be passing this around until we patch the lib to avoid the issue. thanks!

**Comment by clkr** [comment]
@arittr I'm happy to hear this helps. Cheers!

**Comment by fluidsonic** [comment]
Same issue here.

This issue occurs because Mixpanel assumes that its storyboard files are always located in the app's main bundle. This is not the case with the new framework structure introduced with Xcode 6 adopted by CocoaPods 0.36.

In `Mixpanel.m` the storyboard is loaded from the app's main bundle instead of from the bundle where the Mixpanel library is located.
```objc
[UIStoryboard storyboardWithName:@"MPNotification" bundle:nil];
```
should be
```objc
[UIStoryboard storyboardWithName:@"MPNotification" bundle:[NSBundle bundleForClass:Mixpanel.class]];
```

See http://blog.cocoapods.org/CocoaPods-0.36/ (section 'Caveats')
See http://blog.cocoapods.org/Pod-Authors-Guide-to-CocoaPods-Frameworks/

This issue also affects other parts of the Mixpanel library like the survey.

**Comment by clkr** [comment]
@fluidsonic it will be useful to create a pull request if this fixes the issue.

**Comment by fluidsonic** [comment]
@clkr yeah already on it :)

**Comment by fluidsonic** [comment]
Fixed in PR #250
